### PR TITLE
[Feature] CON 81 - Enable text wrap passages

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to this issue**

https://oat-sa.atlassian.net/browse/CON-81

**Other related PRs**

- https://github.com/oat-sa/extension-tao-itemqti/pull/1580
- https://github.com/oat-sa/extension-tao-mediamanager/pull/240

**Story**

As the author, I want to be able to wrap my text around pictures, so that I can produce passages that look like real-life media content.

Currently, I am only able to anchor images AS a character. In order for the text to be able to flow around it, I will need to be able to anchor it TO a character and specify how text is supposed to wrap around it

Required wrap modes:

- Float off
- Float right
- Float left

**Acceptance Criteria**

- Anchoring an image as a character should still be possible
- Text wrapping should not be available when the image is anchored as a character
- Images maintain position within their paragraph when text is added before or after
- Responsive images are still supported if the image is anchored to a character
- Exported items will contain the relevant CSS to ensure the rendering will be correct in any environment
- Exporting and reimporting an item will maintain the anchoring and wrapping

### CODE REVIEW

**Review by gabrielfs7 — APPROVED** [review]
Good job. It seems we just updated the version here. We should also update the tao-core version on manifest before merge/

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -27,7 +27,7 @@
     'label' => 'Development Tools',
     'description' => 'Developer tools that can assist you in creating new extensions, run scripts, destroy your install',
     'license' => 'GPL-2.0',
-    'version' => '6.10.0',
+    'version' => '6.11.0',
     'author' => 'Open Assessment Technologies',
     'requires' => [
         'generis' => '>=11.1.0',
```
Comment (gabrielfs7) [inline manifest.php:33]: We need to update tao-core version too :)

Comment (habemuscode) [reply manifest.php:33]: In requires? mmm then I will do when merge other PR because this can change throw the time

Comment (gabrielfs7) [reply manifest.php:33]: Yes, we release tao-core first, then we update here :)

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by 404passionFruit — APPROVED** [review]
Please don't forget to update tao-core version after release and have a look why Jenkins is not happy

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**